.github/workflows/update_credential_provider_script.yml

```diff
@@ -0,0 +1,38 @@
+name: Update credential provider script
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.7]
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        persist-credentials: false 
+        token: ${{ secrets.GITHUB_TOKEN }}
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r ./test-fixtures/requirements.txt
+    - name: Modify credential provider script
+      run: |
+        python ./Assets/CC_Script/CC_Python_Update.py
+    - name: Commit and push credential provider changed
+      run: |-
+        git diff
+        git diff --quiet || (git add .)
```

Comment:
This is fine for now since you are going to run the github action manually and same to merge it, but in the future once you know the files that change, I would suggest to add them here to prevent from unexpected changes in the PR

Reply:
Totally agreed, need to know final list of files and will update it